requirements-test.txt

```diff
@@ -4,5 +4,9 @@ mock>=1.0.1
 flake8>=2.1.0
 tox>=1.7.0
 twilio==5.4.0
+nose==1.3.7
+djangorestframework==3.3.3
+django-environ==0.4.0
+django-phonenumber-field
```

Comment:
What I would recommend is to specifically put the version number here. To do that you could do `pip freeze` and it will give you the exact line to copy. This way you are sure that the specific version will work.